src/tmdb.py

```
@@ -20,7 +21,13 @@
 from src.console import console
 from src.imdb import get_imdb_info_api
 
-TMDB_API_KEY = config['DEFAULT'].get('tmdb_api', False)
+DEFAULT_CFG: dict[str, Any] = typing_cast(dict[str, Any], config["DEFAULT"])
+TMDB_API_KEY = DEFAULT_CFG.get('tmdb_api', False)
+
+# Validate TMDB API key at import time
+if not TMDB_API_KEY or not isinstance(TMDB_API_KEY, str) or not TMDB_API_KEY.strip():
+    raise ValueError("TMDB API key is missing or invalid. Please set 'tmdb_api' in your config under DEFAULT section.")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)